models.yaml

```diff
@@ -335,3 +357,4 @@ models:
     config_entry_data:
       url: !secret ollama_url
       model: gemma:7b
+
```

Comment:
Please remove this end whitespace, thanks.

Reply:
removed